colorbleed/maya/lib.py

```diff
@@ -1384,7 +1385,8 @@ def set_scene_fps(fps, update=True):
     """
 
     if fps in FLOAT_FPS:
-        unit = "{:f}fps".format(fps)
+        # :.2f means <any digits>.<round to 2 digits>
+        unit = "{:.2f}fps".format(fps)
```

Comment:
Does this still work for `23.976` and `47.952` since they don't match this amount of decimal points as described here?

I think if we remove the forced float formatting this should actually work, so just `"{}fps".format(fps)`